p2p/p2p-core/src/protocol.rs

```diff
@@ -1,9 +1,9 @@
-//! This module defines InternalRequests and InternalResponses. Cuprate's P2P works by translating network messages into an internal
-//! request/ response, this is easy for levin "requests" and "responses" (admin messages) but takes a bit more work with "notifications"
+//! This module defines [`PeerRequest`] and [`PeerResponse`]. Cuprate's P2P crates works by translating network messages into an internal
+//! request/ response enums, this is easy for levin "requests" and "responses" (admin messages) but takes a bit more work with "notifications"
 //! (protocol messages).
 //!
 //! Some notifications are easy to translate, like `GetObjectsRequest` is obviously a request but others like `NewFluffyBlock` are a
-//! bit tri cker. To translate a `NewFluffyBlock` into a request/ response we will have to look to see if we asked for `FluffyMissingTransactionsRequest`
+//! bit tricker. To translate a `NewFluffyBlock` into a request/ response we will have to look to see if we asked for `FluffyMissingTransactionsRequest`,
 //! if we have we interpret `NewFluffyBlock` as a response if not its a request that doesn't require a response.
 //!
 //! Here is every P2P request/ response. *note admin messages are already request/ response so "Handshake" is actually made of a HandshakeRequest & HandshakeResponse
```

Comment:
```suggestion
//! bit tricker. To translate a `NewFluffyBlock` into a request/response we will have to look to see if we asked for [`FluffyMissingTransactionsRequest`],
//! if we have, we interpret `NewFluffyBlock` as a response, if not, it's a request that doesn't require a response.
//!
//! Here is every P2P request/response. *note admin messages are already request/response so "Handshake" is actually made of a HandshakeRequest & HandshakeResponse
```

Reply:
Btw the below `Admin` and `Protocol` table below is not formatted for `cargo doc`.

![image](https://github.com/Cuprate/cuprate/assets/101352116/8c941203-3f29-44b4-ba31-1d0cfa61aac6)

Reply:
Btw x2: maybe `cuprate_wire` should link to here for explanation?

Reply:
This is Cuprate specific, `cuprate_wire` was not built to enable this

Reply:
fixed the formatting as well